deploy/us-east-2-dev/secrets-sealed.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
```

Comment:
A totally different subject for discussion: Have you considered using [`sops`](https://github.com/mozilla/sops) with AWS KMS instead of sealed secrets? That will offer a simpler set up while allowing fine control over encryption keys. It also has built-in support in fluxCD I believe.